src/components/Calculator.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable no-useless-constructor */
+/* eslint-disable react/prefer-stateless-function */
+import React, { Component } from 'react';
+import './calculator.css';
+
+class Calculator extends Component {
+  constructor() {
+    super();
+  }
```

Comment:
- [ ] <b>REQUIRED:</b> This constructor is useless, that is why eslint is complaining about it. Kindly remove it.